deploy/stacks/pipeline.py

```diff
@@ -687,6 +687,8 @@ def set_approval_tests_stage(
                                     f'export TESTDATA=$(aws ssm get-parameter --name /dataall/{target_env["envname"]}/testdata --profile buildprofile --output text --query "Parameter.Value")',
                                     f'export ENVNAME={target_env["envname"]}',
                                     f'export AWS_REGION={target_env["region"]}',
+                                    'export INTEGRATION_TESTS=True',
+                                    f'export TOOLING_ACCOUNT={self.account}',
```

Comment:
* INTEGRATION_TESTS: we can infer wether they are enabled or not by using the presence of the `TESTDATA`
* TOOLING_ACCOUNT: isn't it better if we use an SSM param to figure it out?